client/src/hooks/useAuth.ts

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+const useAuth = () => {
+  const [userId, setUserId] = useState(null);

```

Comment:
replace this 3 state to one state [user, setUser]
becouse we need user data include profile informtion in soon
and i think we dont need ```isAuthorized``` state, just check if user or not